### ISSUE TITLE
Rerunning a submission shows wrong queue on Server Status page

### ISSUE THREAD

**Issue by ObadaS** [codalab/codabench#1595]
Hello, 

It seems like rerunning a submission (either individually or as a mass action), will show the wrong queue on the Server Status page:
![Capture d’écran du 2024-09-25 13-48-54_2](https://github.com/user-attachments/assets/4805d0b5-ab3d-4334-a90e-1863a3f5704e)

All those submissions were rerun, but they are using a private queue, not the default public queue.

This issue is easy to reproduce. 
- Create a queue
- Assign it to your competition
- Run a submission 
- Rerun it from the interface and check the server status page
![image](https://github.com/user-attachments/assets/1c69accf-7b59-494b-ae27-2a27419043e4)

Example:
![image](https://github.com/user-attachments/assets/2444ea22-1279-451e-a3f3-45f4c747ee93)

**Comment by ihsaan-ullah** [comment]
Possible duplicate of #1357

**Comment by ObadaS** [comment]
Oh yea it seems like it's a duplicate.

I can confirm that it uses the assigned queue and not the default one. I tried it by creating a worker on my work PC and connecting it to the competition. It received both jobs.

Someone also recently re-submitted a lot of jobs from their competition. I took a look at our workers and they were not busy, further confirming that this seem to be a visual bug on the Queue shown and the jobs are not being sent to the default Queue.
